docs/relationships/candidate_relationships.md

```diff
@@ -163,7 +163,7 @@ The uninstrumented entity indicates that additional instrumentation can be confi
 
 There are two options for handling `onMiss`:
 
-* **DISCARD**: Discards misses instead of generating an uninstrumented entity.
+* **NO_OP**: Discards misses instead of generating an uninstrumented entity.
```

Comment:
Reported by the bot https://github.com/newrelic/entity-definitions/pull/1261#pullrequestreview-1666446619